app/backend/src/couchers/servicers/donations.py

```diff
@@ -0,0 +1,167 @@
+import json
+import logging
+
+import grpc
+import stripe
+
+from couchers import errors, urls
+from couchers.config import config
+from couchers.db import session_scope
+from couchers.models import Invoice, OneTimeDonation, RecurringDonation, User
+from couchers.sql import couchers_select as select
+from couchers.tasks import send_donation_email
+from couchers.utils import now
+from proto import donations_pb2, donations_pb2_grpc, stripe_pb2_grpc
+from proto.google.api import httpbody_pb2
+
+logger = logging.getLogger(__name__)
+
+
+class Donations(donations_pb2_grpc.DonationsServicer):
+    def __init__(self):
+        self.enabled = config["ENABLE_DONATIONS"]
+
+        if self.enabled:
+            stripe.api_key = config["STRIPE_API_KEY"]
```

Comment:
The stripe Python modules uses global variables, seems it won't be rewritten anytime soon ( https://stripe.com/docs/api/events/types ). I couldn't find any documentation for the Python bindings either.

Better make json requests using the `request` python module. It has [proper docs](https://stripe.com/docs/api/authentication) too.